geomesa-tools/conf/sfts/adsbx/reference.conf

```diff
@@ -0,0 +1,248 @@
+        { name = "Trak"               , type = "Double"                   }
+        { name = "TrkH"               , type = "Boolean"                  }
+        { name = "Type"               , type = "String"                   }
+        { name = "Mdl"                , type = "String"                   }
+        { name = "Man"                , type = "String"                   }
+        { name = "Year"               , type = "Integer"                  }
+        { name = "Cnum"               , type = "String"                   }
+        { name = "Op"                 , type = "String"                   }
+        { name = "OpIcao"             , type = "String"                   }
+        { name = "Sqk"                , type = "Integer"                  }
+        { name = "Vsi"                , type = "Integer"                  }
+        { name = "VsiT"               , type = "String"                   }
+        { name = "WTC"                , type = "String"                   }
+        { name = "Species"            , type = "String"                   }
+        { name = "EngType"            , type = "String"                   }
+        { name = "EngMount"           , type = "String"                   }
+        { name = "Engines"            , type = "String"                   }
+        { name = "Mil"                , type = "Boolean"                  }
+        { name = "Cou"                , type = "String"                   }
+        { name = "From"               , type = "String"                   }
+        { name = "To"                 , type = "String"                   }
+        { name = "Gnd"                , type = "Boolean"                  }
+        { name = "Call"               , type = "String"                   }
+        { name = "CallSus"            , type = "Boolean"                  }
+        { name = "HasPic"             , type = "Boolean"                  }
+        { name = "FlightsCount"       , type = "Integer"                  }
+        { name = "Interested"         , type = "Boolean"                  }
+        { name = "Help"               , type = "Boolean"                  }
+        { name = "Trt"                , type = "String"                   }
+        { name = "TT"                 , type = "String"                   }
+        { name = "Talt"               , type = "Integer"                  }
+        { name = "Ttrk"               , type = "Integer"                  }
+        { name = "Sat"                , type = "Boolean"                  }
+        { name = "PosStale"           , type = "Boolean"                  }
+        { name = "dtg"                , type = "Date"    , default = "true"                 }
+        { name = "geom"               , type = "Point"   , srid = "4326" , default = "true" }
+      ]
+    }
+  } // end sfts
+
+  converters = {
+    adsbx-Base = {
+      type = json
+      options = {
+        validators = ["z-index", "has-geo", "has-dtg"]
```

Comment:
you can just make this `validators = [ "index" ]` - that will work with whatever indices get created, and the has-geo and has-dtg are redundant if there's an index validator